services/web/server/src/simcore_service_webserver/resource_manager/module_setup.py

```diff
@@ -9,31 +9,43 @@
 import logging
 
 from aiohttp import web
-from servicelib.aiohttp.application_setup import ModuleCategory, app_module_setup
+from servicelib.aiohttp.application_setup import (
+    AlreadyInitializedError,
+    ModuleCategory,
+    app_module_setup,
+)
 
+from ..garbage_collector import setup_garbage_collector
+from ..redis import setup_redis
 from .config import APP_CLIENT_SOCKET_REGISTRY_KEY, APP_RESOURCE_MANAGER_TASKS_KEY
-from .garbage_collector import setup_garbage_collector
-from .redis import setup_redis_client
 from .registry import RedisResourceRegistry
-from .settings import assert_valid_config
 
 logger = logging.getLogger(__name__)
 
 
 @app_module_setup(
-    "simcore_service_webserver.resource_manager", ModuleCategory.SYSTEM, logger=logger
+    "simcore_service_webserver.resource_manager", ModuleCategory.ADDON, logger=logger
```

Comment:
just wondering here. the resource manager being an ADDON vs SYSTEM...
it sounds a bit dangerous. The resource manager uses distributed locks to function and is an essential part so that projects do not get removed at the wrong timings. Even the garbage collector indirectly uses it I think. Are you really sure about that change?

Reply:
agreed